campair/campair/View/ToolDictionary/ToolDictionaryMainView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ToolDictionaryMainView.swift
+//  campair
+//
+//  Created by 이가은 on 2022/06/12.
+//
+
+import SwiftUI
+
+struct ToolDictionaryMainView: View {
+    @State private var searchWord : String = ""
+    
+    private let columns: [GridItem] = [
+        GridItem(.flexible(), spacing: 10, alignment: nil),
+        GridItem(.flexible(), spacing: 10, alignment: nil),
+        GridItem(.flexible(), spacing: 10, alignment: nil),
+        GridItem(.flexible(), spacing: 10, alignment: nil)
+    ]
+    
+    var body: some View {
+        NavigationView{
+            VStack(alignment : .leading, spacing: 20){
+                HStack(spacing : 0){
+                    Text("Logo")
+                        .padding(.trailing, 20)
+                    ZStack(alignment: .leading){
+                        Rectangle()
+                            .stroke()
+                            .background(Color.white)
+                            .frame(height: 34)
+                        HStack(spacing : 0){
+                            Image(systemName: "magnifyingglass")
+                                .padding(.horizontal, 10)
+                            TextField("검색어를 입력하세요.", text: self.$searchWord)
+                        }
+                    }
+                }
+                Text("어서와요!(가안)\n장비를 알아보아요")
+                    .bold()
+                    .font(.system(size:25))
```

Comment:
```suggestion
                    .font(.system(size: 25))
```

보통의 Swift 코드 스타일로는 Colon 뒤에 스페이스 하나가 추천됩니다 ~

Reply:
I fixed it thank you !